### ISSUE TITLE
Allow comments on empty blocks

### ISSUE THREAD

**Issue by vegetableman** [eslint/eslint#1414]
``` js
if (a === true) {
   // do something
 } else {
    return b;
}
```

I like to add comments, sort of Todo's when writing code, so as to get back to it later but eslint throws empty block error for the above code based on `no-empty` rule. Could comments be supported or is it a bad practice to allow above code?
